File: WEEK 1/C++/Salesperson's Commission.cpp
#include <iostream>
using namespace std;

int main() {
    double sellingPrice, costOfCar;

    cout << "Enter the selling price of the used car: $";
    cin >> sellingPrice;

    cout << "Enter the cost of the car: $";
    cin >> costOfCar;

    const double BaseCommission = 20.0;
    const double CommissionRate = 0.30;

    double excessPrice = max(0.0, sellingPrice - costOfCar);
    double commission = BaseCommission + (excessPrice * CommissionRate);

    printf("Excess Price: $%.2f\n", excessPrice);
    printf("Commission: $%.2f\n", commission);

    return 0;
}

File: WEEK 1/C#/Salesperson's Commission.cs
using System;

class Program {
    static void Main() {
        Console.Write("Enter the selling price of the used car: R");
        double sellingPrice = double.Parse(Console.ReadLine());

        Console.Write("Enter the cost of the car: R");
        double costOfCar = double.Parse(Console.ReadLine());

        const double BaseCommission = 20.0;
        const double CommissionRate = 0.30;

        double excessPrice = Math.Max(0, sellingPrice - costOfCar);
        double commission = BaseCommission + (excessPrice * CommissionRate);

        Console.WriteLine($"Excess Price: R{excessPrice:F2}");
        Console.WriteLine($"Commission: R{commission:F2}");
    }
}
